### PR TITLE
feat(docs): added inline documentation to experimental packages

### PR DESCRIPTION
**Pull request by sanderson**
Adds inline documentation to the following pacakges:

- experimental/aggregate
- experimental/array
- experimental/bigtable
- experimental/bitwise
- experimental/csv

### Done checklist
- [x] docs/SPEC.md updated **N/A**
- [x] Test cases written **N/A**

### CODE REVIEW

File: stdlib/experimental/csv/csv.flux [hunk 1]
```diff
@@ -1,7 +1,28 @@
...
 import c "csv"
 import "experimental/http"
 
+// from retrieves [annotated CSV](https://docs.influxdata.com/influxdb/latest/reference/syntax/annotated-csv/) **from a URL**.
+// 
+// **Note:** Experimental `csv.from()` is an alternative to the standard
+// `csv.from()` function.
+// 
+// ## Parameters
+// - url: URL to retrieve annotated CSV from.
```
Comment (scbrickley) [inline stdlib/experimental/csv/csv.flux:18]: Shouldn't there also be documentation for the `csv` parameter here?

https://docs.influxdata.com/flux/v0.x/stdlib/csv/from/#csv

Comment (sanderson) [reply stdlib/experimental/csv/csv.flux:18]: @scbrickley This is the `csv.from` in the `experimental/csv` package. It only has a `url` parameter.

- Docs: https://docs.influxdata.com/flux/v0.x/stdlib/experimental/csv/from/
- Source: https://github.com/influxdata/flux/blob/master/stdlib/experimental/csv/csv.flux

Comment (scbrickley) [reply stdlib/experimental/csv/csv.flux:18]: Ah, I missed that. Apologies.

**Review by scbrickley — APPROVED** [review]

File: stdlib/experimental/csv/csv.flux (hunk 1 above)
Comment (sanderson) [reply stdlib/experimental/csv/csv.flux:18]: No worries. Thanks for the review!

**Pull request merged**